examples/basics/src/pages/index.astro

```diff
@@ -12,6 +12,7 @@ import Card from '../components/Card.astro';
 			viewBox="0 0 495 623"
 			fill="none"
 			xmlns="http://www.w3.org/2000/svg"
+      aria-hidden="true" focusable="false"
```

Comment:
```suggestion
      aria-hidden="true"
```

> `focusable="false"` is also used to ensure Internet Explorer won’t allow the Tab key to navigate into the SVG.

I don't think we need to be concerned with IE support since it's EOL.